src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -105,6 +105,7 @@ public static void ZeroLengthContent_RoundTrip()
                 }
                 catch (CryptographicException) when (PlatformDetection.IsFullFramework) // Expected on full FX
                 {
+                    return;
```

Comment:
https://referencesource.microsoft.com/#System.Security/system/security/cryptography/pkcs/envelopedpkcs7.cs,200

Desktop says if 0 throw.

http://source.dot.net/#System.Security.Cryptography.Pkcs/System/Security/Cryptography/Pkcs/EnvelopedCms.cs,95

Core doesn't.

@AtsushiKan I don't suppose you remember why? :smile: